source/jormungandr/jormungandr/parking_space_availability/car/common_car_park_provider.py

```diff
@@ -60,6 +61,15 @@ def __init__(self, url, operators, dataset, timeout, feed_publisher, **kwargs):
 
         self.api_key = kwargs.get('api_key')
 
+        self.shape = None
+        arg_geometry = kwargs.get('geometry')
+        if arg_geometry:
+            try:
+                self.shape = shapely.geometry.shape(arg_geometry)
```

Comment:
You have to check the validity of the shape: https://shapely.readthedocs.io/en/latest/manual.html#object.is_valid

see https://github.com/CanalTP/navitia/pull/2658

Reply:
:+1: